src/test/kotlin/org/rust/ide/annotator/fixes/AddStructFieldsFixTest.kt

```diff
@@ -103,7 +103,7 @@ class AddStructFieldsFixTest : RsAnnotatorTestBase(RsExpressionAnnotator::class)
                 0: 92,
                 1: 0/*caret*/,
                 2: 92,
-                3: 0
+                3: 0,
```

Comment:
I think that we should add trailing comma only if we just generated the last field, and shouldn't touch the last field if it already exists and we generated some fields in the middle

Reply:
[04e4535](https://github.com/intellij-rust/intellij-rust/pull/9515/commits/04e4535008cbe3c94f795c230597395597f7829e)